app/Commands/NewCommand.php

```diff
@@ -56,56 +68,121 @@ public function buildSignatureOption($option)
 
     public function handle()
     {
-        app()->bind('console', function () {
-            return $this;
-        });
-
-        app(DisplayLamboWelcome::class)();
+        $this->makeAndInvoke(DisplayLamboWelcome::class);
 
         if (! $this->argument('projectName')) {
-            app(DisplayHelpScreen::class)();
+            $this->makeAndInvoke(DisplayHelpScreen::class);
             exit;
         }
 
-        $this->alert('Creating a Laravel app ' . $this->argument('projectName'));
+        $this->setConfig();
 
-        app(SetConfig::class)();
+        $this->consoleWriter->section("Creating a new Laravel app '{$this->argument('projectName')}'");
 
         try {
-            app(VerifyPathAvailable::class)();
-
-            app(VerifyDependencies::class)(['laravel', 'git', 'valet']);
-
-            app(RunLaravelInstaller::class)();
-
-            app(OpenInEditor::class)();
-
-            app(CustomizeDotEnv::class)();
-
-            $this->info(app(CreateDatabase::class)());
-
-            app(GenerateAppKey::class)();
-
-            app(ConfigureFrontendFramework::class)();
-
-            app(InitializeGitRepo::class)();
-
-            app(InstallNpmDependencies::class)();
-
-            app(CompileAssets::class)();
-
-            app(RunAfterScript::class)();
-
-            app(ValetLink::class)();
-
-            app(ValetSecure::class)();
+            $this->makeAndInvoke(ValidateConfiguration::class);
```

Comment:
A note: I think we said we didn't need `makeAndInvoke`, right? I'll merge, but just for future notes.

Reply:
Yes. I've actually completed the work to remove it but I've been doing that (and other requests) in a jrs/lambo-beta3 branch. I figured that I shouldn't add any more to this PR or we will be in an endless review loop.

Once this is released I will PR the new work.